bundles/org.openhab.binding.myenergi/src/main/java/org/openhab/binding/myenergi/internal/handler/MyenergiZappiHandler.java

```diff
@@ -0,0 +1,275 @@
+import static org.openhab.binding.myenergi.internal.MyenergiBindingConstants.ZAPPI_CHANNEL_CLAMP_NAME_1;
+import static org.openhab.binding.myenergi.internal.MyenergiBindingConstants.ZAPPI_CHANNEL_CLAMP_NAME_2;
+import static org.openhab.binding.myenergi.internal.MyenergiBindingConstants.ZAPPI_CHANNEL_CLAMP_NAME_3;
+import static org.openhab.binding.myenergi.internal.MyenergiBindingConstants.ZAPPI_CHANNEL_CLAMP_NAME_4;
+import static org.openhab.binding.myenergi.internal.MyenergiBindingConstants.ZAPPI_CHANNEL_CLAMP_NAME_5;
+import static org.openhab.binding.myenergi.internal.MyenergiBindingConstants.ZAPPI_CHANNEL_CLAMP_NAME_6;
+import static org.openhab.binding.myenergi.internal.MyenergiBindingConstants.ZAPPI_CHANNEL_CLAMP_POWER_1;
+import static org.openhab.binding.myenergi.internal.MyenergiBindingConstants.ZAPPI_CHANNEL_CLAMP_POWER_2;
+import static org.openhab.binding.myenergi.internal.MyenergiBindingConstants.ZAPPI_CHANNEL_CLAMP_POWER_3;
+import static org.openhab.binding.myenergi.internal.MyenergiBindingConstants.ZAPPI_CHANNEL_CLAMP_POWER_4;
+import static org.openhab.binding.myenergi.internal.MyenergiBindingConstants.ZAPPI_CHANNEL_CLAMP_POWER_5;
+import static org.openhab.binding.myenergi.internal.MyenergiBindingConstants.ZAPPI_CHANNEL_CLAMP_POWER_6;
+import static org.openhab.binding.myenergi.internal.MyenergiBindingConstants.ZAPPI_CHANNEL_CONSUMED_POWER;
+import static org.openhab.binding.myenergi.internal.MyenergiBindingConstants.ZAPPI_CHANNEL_DIVERTED_POWER;
+import static org.openhab.binding.myenergi.internal.MyenergiBindingConstants.ZAPPI_CHANNEL_DIVERTER_PRIORITY;
+import static org.openhab.binding.myenergi.internal.MyenergiBindingConstants.ZAPPI_CHANNEL_GENERATED_POWER;
+import static org.openhab.binding.myenergi.internal.MyenergiBindingConstants.ZAPPI_CHANNEL_GRID_POWER;
+import static org.openhab.binding.myenergi.internal.MyenergiBindingConstants.ZAPPI_CHANNEL_LAST_COMMAND_STATUS;
+import static org.openhab.binding.myenergi.internal.MyenergiBindingConstants.ZAPPI_CHANNEL_LAST_UPDATED_TIME;
+import static org.openhab.binding.myenergi.internal.MyenergiBindingConstants.ZAPPI_CHANNEL_MINIMUM_GREEN_LEVEL;
+import static org.openhab.binding.myenergi.internal.MyenergiBindingConstants.ZAPPI_CHANNEL_PLUG_STATUS;
+import static org.openhab.binding.myenergi.internal.MyenergiBindingConstants.ZAPPI_CHANNEL_SMART_BOOST_CHARGE;
+import static org.openhab.binding.myenergi.internal.MyenergiBindingConstants.ZAPPI_CHANNEL_SMART_BOOST_DURATION;
+import static org.openhab.binding.myenergi.internal.MyenergiBindingConstants.ZAPPI_CHANNEL_SUPPLY_FREQUENCY;
+import static org.openhab.binding.myenergi.internal.MyenergiBindingConstants.ZAPPI_CHANNEL_SUPPLY_VOLTAGE;
+import static org.openhab.binding.myenergi.internal.MyenergiBindingConstants.ZAPPI_CHANNEL_TIMED_BOOST_CHARGE;
+import static org.openhab.binding.myenergi.internal.MyenergiBindingConstants.ZAPPI_CHANNEL_TIMED_BOOST_DURATION;
```

Comment:
Wildcard imports are allowed for static imports like this:
```suggestion
import static org.openhab.binding.myenergi.internal.MyenergiBindingConstants.*;
```